bytessource.go

```diff
@@ -0,0 +1,65 @@
+package fuzz
+
+import (
+	"encoding/binary"
+	"math/rand"
+)
+
+// byteSource implements rand.Source64 determined by a slice of bytes. The random numbers are
```

Comment:
byteSource vs bytesSource, it doesn't matter which you pick but they should match :)